01 Operator_Overloading/01 OperatorOverloading/OperatorOverloading.DBL/JsonParser.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using OperatorOverloading.Parse;
+
+namespace OperatorOverloading.DBL
+{
+   public class JsonParser
+    {
+        const string SOURCE = "source";
+        private static string  _sourceCurrency;
+       
+        public static Dictionary<string, double> ParseJson(string jsonString,string sourceCurrency) 
+        {
+            Dictionary<String, double> rateDictionary = new Dictionary<String, double>();
+            _sourceCurrency = sourceCurrency;
```

Comment:
Why do i need to assign local variable the source currency, when I am passing it in function.